concepts/match-basics/about.md

```diff
@@ -138,11 +176,36 @@ fn toss(value: u8) {
 }
 
 fn main() {
-let x: u8 = 2;
-let y: u8 = 3;
-let z: u8 = 5;
-toss(x); // prints "you won!".
-toss(y); // prints "you can roll again!".
-toss(z); // prints "you lost...".
+    let x: u8 = 2;
+    let y: u8 = 3;
+    let z: u8 = 5;
+
+    toss(x); // prints "you won!".
+    toss(y); // prints "you can roll again!".
+    toss(z); // prints "you lost...".
 }
 ```
+
+## Matching Tuples
+
+Tuples can also be matched against a set of patterns. These patterns are designed similarly as a tuple.
+
+```rust
+...
+#[derive(Drop)]
+enum DayType {
+    Week,
+    Weekend,
+    Holiday
+}
+
+fn vending_machine_accept(c: (DayType, Color)) -> bool {
+    match c {
+        (DayType::Week, _) => true, // accepts all colors during week day
+        (_, Color::Red) | (_, Color::Green) => true, // accepts only Red or Green color for all specified days
+        (_, _) => false, // returns false for other instances
+    }
+}
+```
+
+Using `(_, _)` is unnecessary as it performs the same functionality as using a single wildcard: `_ =>` syntax.
```

Comment:
```suggestion
In the last match arm, it doesn't matter if we use `(_, _)` or just the `_` wildcard to match any day and color combination, because they're functionally the same.
```
Reworded to be a bit more clear.